tests/unittests/tests-ipv6_nc/tests-ipv6_nc.c

```diff
@@ -0,0 +1,336 @@
+    ng_ipv6_nc_remove(DEFAULT_TEST_NETIF, &addr);
+    TEST_ASSERT_NOT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &exp_addr));
+    TEST_ASSERT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr));
+}
+
+static void test_ipv6_nc_remove__no_entry_addr2(void)
+{
+    ng_ipv6_addr_t addr = NG_IPV6_ADDR_UNSPECIFIED;
+    ng_ipv6_addr_t exp_addr = DEFAULT_TEST_IPV6_ADDR;
+
+    test_ipv6_nc_add__success(); /* adds DEFAULT_TEST_IPV6_ADDR to DEFAULT_TEST_NETIF */
+    ng_ipv6_nc_remove(DEFAULT_TEST_NETIF, &addr);
+    TEST_ASSERT_NOT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &exp_addr));
+    TEST_ASSERT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr));
+}
+
+static void test_ipv6_nc_remove__success(void)
+{
+    ng_ipv6_addr_t addr = DEFAULT_TEST_IPV6_ADDR;
+
+    test_ipv6_nc_add__success(); /* adds DEFAULT_TEST_IPV6_ADDR to DEFAULT_TEST_NETIF */
+    TEST_ASSERT_NOT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr));
+    ng_ipv6_nc_remove(DEFAULT_TEST_NETIF, &addr);
+    TEST_ASSERT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr));
+}
+
+static void test_ipv6_nc_get__empty(void)
+{
+    ng_ipv6_addr_t addr = DEFAULT_TEST_IPV6_ADDR;
+
+    TEST_ASSERT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr));
+}
+
+static void test_ipv6_nc_get__different_if(void)
+{
+    ng_ipv6_addr_t addr = DEFAULT_TEST_IPV6_ADDR;
+
+    test_ipv6_nc_add__success(); /* adds DEFAULT_TEST_IPV6_ADDR to DEFAULT_TEST_NETIF */
+
+    TEST_ASSERT_NULL(ng_ipv6_nc_get(OTHER_TEST_NETIF, &addr));
+}
+
+static void test_ipv6_nc_get__different_addr(void)
+{
+    ng_ipv6_addr_t addr = OTHER_TEST_IPV6_ADDR;
+
+    test_ipv6_nc_add__success(); /* adds DEFAULT_TEST_IPV6_ADDR to DEFAULT_TEST_NETIF */
+
+    TEST_ASSERT_NULL(ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr));
+}
+
+static void test_ipv6_nc_get__success1(void)
+{
+    ng_ipv6_addr_t addr = DEFAULT_TEST_IPV6_ADDR;
+    ng_ipv6_nc_t *entry = NULL;
+
+    test_ipv6_nc_add__success(); /* adds DEFAULT_TEST_IPV6_ADDR to DEFAULT_TEST_NETIF */
+
+    TEST_ASSERT_NOT_NULL((entry = ng_ipv6_nc_get(DEFAULT_TEST_NETIF, &addr)));
+    TEST_ASSERT_EQUAL_INT(DEFAULT_TEST_NETIF, entry->iface);
+    TEST_ASSERT(ng_ipv6_addr_equal(&(entry->ipv6_addr), &addr));
+    TEST_ASSERT_EQUAL_STRING(TEST_STRING4, (char *)entry->l2_addr);
+    TEST_ASSERT_EQUAL_INT(sizeof(TEST_STRING4), entry->l2_addr_len);
+}
+
+static void test_ipv6_nc_get__success2(void)
+{
+    ng_ipv6_addr_t addr = DEFAULT_TEST_IPV6_ADDR;
+    ng_ipv6_nc_t *entry = NULL;
+
+    test_ipv6_nc_add__success(); /* adds DEFAULT_TEST_IPV6_ADDR to DEFAULT_TEST_NETIF */
+
+    TEST_ASSERT_NOT_NULL((entry = ng_ipv6_nc_get(KERNEL_PID_UNDEF, &addr)));
+    TEST_ASSERT_EQUAL_INT(DEFAULT_TEST_NETIF, entry->iface);
+    TEST_ASSERT(ng_ipv6_addr_equal(&(entry->ipv6_addr), &addr));
+    TEST_ASSERT_EQUAL_STRING(TEST_STRING4, (char *)entry->l2_addr);
+    TEST_ASSERT_EQUAL_INT(sizeof(TEST_STRING4), entry->l2_addr_len);
+    TEST_ASSERT_EQUAL_INT(0, entry->flags);
```

Comment:
No, this is an interface global search (see https://github.com/RIOT-OS/RIOT/pull/2461/files#diff-bfdde70ccc3110439add7db00ff1edd6R200). But this test is missing in `test_ipv6_nc_add__success();`